glide.yaml

```diff
@@ -1,3 +1,10 @@
-package: github.com/deis/pkg/time
+package: github.com/deis/controller-sdk-go
 import:
-- package: github.com/deis/pkg/time
+- package: github.com/deis/pkg
+  subpackages:
+  - time
+- package: github.com/goware/urlx
+- package: github.com/PuerkitoBio/purell
```

Comment:
@Joshua-Anderson I'm assuming this and the below package are dependencies of `github.com/goware/urlx`? If so, did `glide up` not pick those deps up and put them into the `glide.lock`?